src/main/pages/setup-openshift/openshift-config.adoc

```diff
@@ -206,7 +205,19 @@ CHE_KEYCLOAK_OIDC__PROVIDER=<base URL of the OIDC provider that provides a confi
 If the optional http://openid.net/specs/openid-connect-core-1_0.html#AuthRequest[`nonce` OpenId request parameter] is not supported, the following environment variable should be added:
 
 ----
-CHE_KEYCLOAK.USE__NONCE=FALSE
+CHE_KEYCLOAK_USE__NONCE=FALSE
+----
+
+By default the OIDC authentication in Che requires using redirect URLs with wildcards. However, if the alternate OIDC provider only supports a list of fixed redirect URLs, then you should use the following environment variable to switch to fixed redirect URL mode:
+
+----
+CHE_KEYCLOAK_USE__FIXED__REDIRECT__URLS=TRUE
+----
+
+In this case the 2 redirect URLs that should be registered in the OIDC provider configuration are:
```

Comment:
_In this case, the two redirect URLs that should be registered in the OIDC provider configuration are:_

Reply:
Fixed in commit https://github.com/eclipse/che-docs/pull/509/commits/0d2262ef42c188c6305016283ad5f5b347b360c7